cirq/experiments/google_v2_supremacy_circuit_test.py

```diff
@@ -44,6 +45,7 @@ def test_google_v2_supremacy_circuit():
     assert circuit.operation_at(qubits[14], 7).gate == ops.CZ
 
 
+@cirq.testing.skip_if_module_not_exists(module="cirq.google")
```

Comment:
agreed - no clue how to do that yet - probably when we'll move cirq.experiments to cirq.qcvv we'll create a cirq.google.qcvv as well? That kind of cleans up the dependencies but fragments the package.

Reply:
`cirq.google.qcvv` feels right to me. There will almost certainly be other platform-specific qcvv experiments. as those experiments will need platform-specific spaces to live.

Reply:
Yes, that is true. 

A thought here: a fragmented qcvv package could benefit from having a single entrypoint. I imagine that we could have `cirq_qcvv` (a new top level module) that could pull in optionally the cross-platfrom, the google-specific and other platform specific qcvv methods, then a user could just do `pip install cirq_qcvv` and have access to all these methods from a single namespace. @mpharrigan

Reply:
Follow-up after cirq cynque - this will be done separately, @mpharrigan already has plans to rename and move these functions to ReCirq. xref https://github.com/quantumlib/Cirq/issues/3775